codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoderTest.java

```diff
@@ -200,20 +201,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
     }
 
     @Test
-    public void dataWriteAfterGoAwayShouldFail() throws Exception {
```

Comment:
I think that is OK.  We just want to enforce the behavior that *should* be implemented.  The fact that we don't check for GOAWAY in these methods is what we are testing.  So testing:
1. We can send HEADERS and DATA after a GO_AWAY is sent (mocking the appropriate methods to make it look like we sent a goaway).
2.We can not send a HEADERS and DATA after a GO_AWAY is received AND the `lastStreamId` > what was in the GO_AWAY frame we received.